script/release.sh

```diff
@@ -51,10 +69,15 @@ function build_project() {
 	# Add -a to go build flags to make sure it links against
 	# the provided libseccomp, not the system one (otherwise
 	# it can reuse cached pkg-config results).
-	make -C "$root" PKG_CONFIG_PATH="${prefix}/lib/pkgconfig" COMMIT_NO= EXTRA_FLAGS="-a" EXTRA_LDFLAGS="${ldflags}" static
-	rm -rf "$prefix"
+	make -C "$root" PKG_CONFIG_PATH="${LIBSECCOMP_PREFIX}/lib/pkgconfig" COMMIT_NO= EXTRA_FLAGS="-a" EXTRA_LDFLAGS="${ldflags}" static
```

Comment:
Just wanted to point out that the hardcoding of `/lib/pkgconfig` as the path means this script hasn't worked on my machine for several releases now (I manually patch it to use `/lib64/...` which is where the pkgconfig descriptions get installed under openSUSE). I've been meaning to fix it properly (adding `--libdir` to `configure` should work) but since you said you're going to rework `release.sh` can you please fix this as well? Thanks.

Reply:
That's weird; `./configure --help says`:

>   --libdir=DIR            object code libraries [EPREFIX/lib]

so I can't understand how this can be changed to lib64. Guess it's time to add opensuse to my VM collection.

(half an hour later)

I guess one does not know autotools good enough, ever.

Maybe you can just can do `unset SITE_CONFIG`...

Anyway, let me patch it.

Reply:
Fixed by #3200